packages/shared/src/components/PromotionalBanner.tsx

```diff
@@ -3,12 +3,28 @@ import classNames from 'classnames';
 import XIcon from './icons/Close';
 import { Button } from './buttons/Button';
 import { isTesting } from '../lib/constants';
-import { BannerData } from '../graphql/banner';
+import { BannerCustomTheme, BannerData, BannerTheme } from '../graphql/banner';
+import { Theme } from './utilities';
 
 interface PromotionalBannerProps {
   bannerData: BannerData;
   setLastSeen: (value: Date) => Promise<void>;
 }
+
+const classNamesByTheme: Record<BannerTheme, string[]> = {
+  [BannerCustomTheme.cabbageOnion]: ['bg-theme-color-cabbage bg-gradient-to-b', 'text-white', 'bg-white text-pepper-90'],
```

Comment:
@rebelchris is `bg-gradient-to-b` correct for a gradient bottom?

Reply:
Yep, though these ones are not yet whitelisted from the rule if I'm not mistaken. We can add this manually here:
https://github.com/dailydotdev/apps/blob/master/packages/eslint-rules/lib/rules/no-custom-color.js#L13

I've seen multiple introductions to new classes like this. In the future, probably would need to make our custom rule accept a prefix and automatically whitelist everything that follows.
Something like `bg-gradient-` will white list:
`bg-gradient-to-b`,
`bg-gradient-to-br`,
etc.